src/pyTOPP/src/pyopenms/pytopp/ctdsupport.py

```
@@ -0,0 +1,813 @@
+
+    def __init__(self, model: CTDModel):
+        self.model = model
+
+    # ---------- public API -------------------------------------------------
+
+    def parse(
+        self,
+        argv: list[str],
+        *,
+        mode: str = "pure",
+        openms_param: Optional[Any] = None,
```

Ruff (0.14.1)

Undefined name `Optional`

(F821)